src/api/app/controllers/attribute_controller.rb

```diff
@@ -302,4 +271,25 @@ def find_attribute_container
     params[:namespace] = name_parts[0]
     params[:name] = name_parts[1]
   end
+
+  private
+
+  def ensure_namespace
```

Comment:
who is this related with rubocop? It seems like it is in the wrong commit

Reply:
I squashed now everything in one commit